Code/Launcher/BFISH.CPP

```diff
@@ -339,12 +339,12 @@ int BlowfishEngine::Decrypt(void const * cyphertext, int length, void * plaintex
  *                                                                                             *
  * OUTPUT:  none                                                                               *
  *                                                                                             *
- * WARNINGS:   The source and destination buffers must be 8 bytes long.                        *
+ * WARNINGS:   The source and destination buffers must be 8 bytes int.                        *
```

Comment:
if this comment is to believed, some parts of the code may need to be changed to be `uint64_t` or similar